src/vs/base/parts/quickinput/common/quickInput.ts

```diff
@@ -18,6 +18,8 @@ export interface IQuickPickItemHighlights {
 	detail?: IMatch[];
 }
 
+export type BaseQuickPickItem = IQuickPickSeparator | IQuickPickItem;
```

Comment:
Base implies to me an abstract class, how about `QuickPickItem` since in the API a separator is considered a type of quick pick item?

https://github.com/microsoft/vscode/blob/eaa648574d2a61140221a5e0dd3eea56ae11e60e/src/vscode-dts/vscode.d.ts#L1730-L1734